.coveragerc

```diff
@@ -1,5 +1,12 @@
-[report]
 # Don't worry about coverage for the grid GUI (for now)
+[run]
 omit =
+    armi/cli/gridGui.py
     armi/utils/gridEditor.py
+    armi/utils/tests/test_gridGui.py
+
+[report]
+omit =
     armi/cli/gridGui.py
+    armi/utils/gridEditor.py
+    armi/utils/tests/test_gridGui.py
```

Comment:
Even with the `ARMI_GUI_TESTS` variable thing, we should leave these so the coverage report doesnt make us too sad

Reply:
👍